packages/react-components/src/components/SendBox.tsx

```diff
@@ -182,6 +215,16 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
     setTextValue(newValue);
   };
 
+  /* @conditional-compile-remove-from(stable): FILE_SHARING */
+  const renderFileUploadErrorMessage: JSX.Element = useMemo(() => {
+    const fileUploads: ActiveFileUploads[] = props.activeFileUploads || [];
+    const latestError = fileUploads.filter((fileUpload) => fileUpload.errorMessage).pop();
+    if (latestError) {
+      return <Stack className={mergeStyles(errorBarStyle(theme))}>{latestError.errorMessage}</Stack>;
+    }
+    return <Stack></Stack>;
```

Comment:
I would recommend returning `<></>` instead of adding an unnecessary Stack

Reply:
+1 (I would even go so far as to return `undefined` to make it clear this may produce nothing, but I think both `<></>` and `undefined` will result in no DOM being created)